pkgs/development/libraries/ndn-cxx/default.nix

```diff
@@ -1,24 +1,36 @@
-{ stdenv, fetchFromGitHub, openssl, doxygen
-, boost, sqlite, pkgconfig, python, pythonPackages, wafHook }:
-let
-  version = "0.6.3";
-in
-stdenv.mkDerivation {
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, wafHook
+, openssl
+, doxygen
+, boost
+, sqlite
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
   pname = "ndn-cxx";
-  inherit version;
+  version = "0.7.1";
+
   src = fetchFromGitHub {
     owner = "named-data";
     repo = "ndn-cxx";
-    rev = "a3bf4319ed483a4a6fe2c96b79ec4491d7217f00";
-    sha256 = "076jhrjigisqz5n8dgxwd5fhimg69zhm834m7w9yvf9afgzrr50h";
+    rev = "${pname}-${version}";
+    sha256 = "1lcaqc79n3d9sip7knddblba17sz18b0w7nlxmj3fz3lb3z9qd51";
   };
+
   nativeBuildInputs = [ pkgconfig wafHook ];
-  buildInputs = [ openssl doxygen boost sqlite python pythonPackages.sphinx];
+
+  buildInputs = [ openssl doxygen boost sqlite python3 python3Packages.sphinx ];
```

Comment:
Is sphinx a runtime dependency? I thought it only generates docs.

Reply:
Moved to nativeBuildInputs (together with python3 and doxygen) in 50bf5d66741919888d9fb72a0c7b6633a9969629